redocly.yaml

```diff
@@ -0,0 +1,6 @@
+extends:
+  - recommended
+rules:
+  no-invalid-media-type-examples:
+    severity: off
+  info-license-url: off
```

Comment:
no-invalid-media-type-examples is a linting rule that should check all the examples against the  schema specification.
Where we have refs to example response files, the redocly library fails to expand the ref and fails the validation (known issue https://github.com/Redocly/redocly-cli/issues/919)

However, we have our own validation (validate_oas_examples.py) that validates all our examples against the schema